src/test/java/core/basesyntax/SalaryInfoTest.java

```diff
@@ -43,7 +43,7 @@
     };
 
     private static final String[] secondScriptArray = {
-            "13.07.2019 John 60 50",
+            "13.07.2019 John 60 50",//-
```

Comment:
```suggestion
            "13.07.2019 John 60 50",
```
please don't change tests